app/about_us/OurCrew.js

```diff
@@ -0,0 +1,50 @@
+import data from "/data/crewData.json";
+const OurCrew = () => {
+  // TASK - React 1 week 1
+  // Create the "Our Crew section"
+  // Use the descriptions provided in /app/about_us/README.md
+  // Use the pictures from /public/crew
+  // Some inspiration ideas found in /data/inspiration_about_us
+  return (
+    <div>
+      <p>{data.intro}</p>
+
+      <div
+        style={{
+          display: "flex",
+          gap: "2rem",
+          justifyContent: "center",
+          margin: "2rem 0",
+          flexWrap: "wrap",
+        }}
+      >
+        {data.crew
+          .filter((member) => member.image)
+          .map((member, index) => (
+            <div key={index} style={{ textAlign: "center" }}>
```

Comment:
Never use index as the key. React uses the key to optimize rendering. Use something unique, like name.

```suggestion
            <div key={member.name} style={{ textAlign: "center" }}>
```